### ISSUE TITLE
Slave GM crash during master GM attempt to stop it

### ISSUE THREAD

**Issue by dumbbell** [rabbitmq/rabbitmq-server#267]
Jenkins seems to reproduce the following issue on a daily-basis:

* The master node wants to stop all slaves:

 ```
=WARNING REPORT==== 13-Aug-2015::20:00:22 ===
Mirrored queue 'amq.gen-1FZKSMhtnXtB65D-zWh_jw' in vhost '/': Stopping all nodes on master shutdown since no synchronised slave is available
```
* The slave node's GM crashes:

 ```
=ERROR REPORT==== 13-Aug-2015::20:00:22 ===
** Generic server <0.334.0> terminating
** Last message in was {'DOWN',#Ref<0.0.2.787>,process,<8771.5295.14>,
                               shutdown}
** When Server state == {state,
                            {3,<0.334.0>},
                            {{0,<8771.5295.14>},#Ref<0.0.2.786>},
                            {{0,<8771.5295.14>},#Ref<0.0.2.787>},
                            {resource,<<"/">>,queue,
                                <<"amq.gen-1FZKSMhtnXtB65D-zWh_jw">>},
                            rabbit_mirror_queue_slave,
                            {4,
                             [{{0,<8771.5295.14>},
                               {view_member,
                                   {0,<8771.5295.14>},
                                   [],
                                   {3,<0.334.0>},
                                   {3,<0.334.0>}}},
                              {{3,<0.334.0>},
                               {view_member,
                                   {3,<0.334.0>},
                                   [],
                                   {0,<8771.5295.14>},
                                   {0,<8771.5295.14>}}}]},
                            0,
                            [{{0,<8771.5295.14>},
                              {member,
                                  {[{12,{delete_and_terminate,shutdown}}],[]},
                                  12,11}},
                             {{3,<0.334.0>},{member,{[],[]},0,0}}],
                            [<0.332.0>],
                            {[],[]},
                            [],0,undefined,
                            #Fun<rabbit_misc.execute_mnesia_transaction.1>,
                            {true,{shutdown,ring_shutdown}}}
** Reason for termination ==
** {bad_return_value,
       {error,
           {{badmatch,{error,not_found}},
            [{gm,'-record_dead_member_in_group/3-fun-1-',2,
                 [{file,"src/gm.erl"},{line,1143}]},
             {mnesia_tm,apply_fun,3,[{file,"mnesia_tm.erl"},{line,833}]},
             {mnesia_tm,execute_transaction,5,
                 [{file,"mnesia_tm.erl"},{line,808}]},
             {rabbit_misc,'-execute_mnesia_transaction/1-fun-0-',1,
                 [{file,"src/rabbit_misc.erl"},{line,526}]},
             {worker_pool_worker,'-run/2-fun-0-',3,
                 [{file,"src/worker_pool_worker.erl"},{line,80}]}]}}}
```

This could be a followup of #224.

**Comment by dumbbell** [comment]
This crash didn't occur since #465 / #466 was fixed.

**Comment by michaelklishin** [comment]
I think this can be closed, we haven't seen this in a while since #465/#466.
